src/modules/netinstall/README.md

```diff
@@ -26,3 +29,33 @@ The file is composed of a list of entry, each describing one group. The keys *na
 
 Two more keys are supported, *hidden* (if true, do not show the group on the page) and *selected* (if true, display the group as selected). Both default to false if not present.
 If both keys are set to true for the same group, you are basically creating a "default" group of packages which will always be installed in the user's system.
+
+## Configuration of the module
+Here is the set of instructions to have the module work in your Calamares. As of July 2016, this has been successfully 
+tested using the live installation of Chakra Fermi.
+
+In the *settings.conf* file, decide where the **netinstall** page should be displayed. I put it just after the
+**welcome** page, but any position between that and just before **partition** should make no difference.
+
+If not present, add the **packages** job in the **exec** list. This is the job that calls the package manager
+to install packages. Make sure it is configured to use the correct package manager for your distribution; this
+is configured in src/modules/packages/packages.conf.
+
+The exec list should be:
+
+  - unpackfs
+  - networkcfg
+  - packages
+
+**unpackfs** creates the chroot where the installation is performed, and unpacks the root image with the filesystem
+structure; **networkcfg** set ups a working network in the chroot; and finally **packages** can install packages
+in the chroot.
+
+## Common issues
+If launching the package manager command returns you negative exit statuses and nothing is actually invoked, this
+is likely an error in the setup of the chroot; check that the parameter **rootMountPoint** is set to the correct
+value in the Calamares configuration.
+
+If the command is run, but exits with error, check that the network is working in the chroot. Make sure /etc/resolv.conf
+exists and that it's not empty.
+
```

Comment:
We might want to point out that the netinstall module won't work if there's no internet connection. In that case, internet connectivity should be a mandatory requirement in `welcome.conf`.